### PR TITLE
CAF-3029 Enhancements to Audit Monkey for Kibana Demo

### PR DESCRIPTION
**Pull request by glucy**
Enchancement to the Audit Monkey to provide the Demo Monkey which generates data and populates Elasticsearch with a real world style mixture of Audit Events.

### CODE REVIEW

File: caf-audit-monkey/src/main/java/com/github/cafaudit/auditmonkey/DemoMonkey.java [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+    }
+    
+    /**
+     * Populate docs array with 8 pre-configured docs
+     */
+    private void initialiseDocArray()
+    {
+        docs[0] = new Doc(1234500001, "A New Hope", "pdf");
+        docs[1] = new Doc(1234500002, "Empire Strikes Back", "msg");
+        docs[2] = new Doc(1234500003, "Retrun of the Jedi", "txt");
```
Comment (CJComac) [inline caf-audit-monkey/src/main/java/com/github/cafaudit/auditmonkey/DemoMonkey.java]: Jedi don't run, they return ;)

File: caf-audit-monkey/src/main/java/com/github/cafaudit/auditmonkey/DemoMonkey.java [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+        docs[3] = new Doc(1234500004, "The Hobbit", "pdf");
+        docs[4] = new Doc(1234500005, "Fellowship of the Ring", "txt");
+        docs[5] = new Doc(1234500006, "Twin Towers", "msg");
```
Comment (CJComac) [inline caf-audit-monkey/src/main/java/com/github/cafaudit/auditmonkey/DemoMonkey.java]: "Two Towers" :)

Comment (glucy) [reply caf-audit-monkey/src/main/java/com/github/cafaudit/auditmonkey/DemoMonkey.java]: 👍

File: caf-audit-monkey/src/main/java/com/github/cafaudit/auditmonkey/DemoMonkey.java (hunk 1 above)
Comment (glucy) [reply caf-audit-monkey/src/main/java/com/github/cafaudit/auditmonkey/DemoMonkey.java]: 👍

**Review by mulhollandc — APPROVED** [review]

**Pull request merged**